packages/sanity/src/core/store/_legacy/document/document-pair/utils/fetchFeatureToggle.ts

```diff
@@ -26,7 +26,6 @@ function createFeatureToggle(client: SanityClient) {
       client.observable
         .request({
           uri: `/data/actions/${dataset}`,
-          withCredentials: true,
```

Comment:
@bjoerge Can you confirm this to be safe? Any idea why we passed `withCredentials: true` here explicitly? (this should have implicitly been the case unless a client not based on the source's `getClient`/`useClient` was used)

Reply:
Probably just old habit/copy paste – should be safe to remove!